src/stores/TxStore.js

```diff
@@ -105,8 +100,6 @@ class TxStore {
     web3.eth.getTransactionReceipt(hash, (error, res) => {
       if(res && res.blockNumber){
         if(res.status === '0x1'){
```

Comment:
consider to compare the number `1` instead of the string in hexadecimal `0x1`. The status code is not standardized well that is why some clients could return `0x01` (`rskj` for example).

Reply:
Done